influxdb/README.md

```diff
@@ -245,156 +583,11 @@ $ docker run --rm \
       -e INFLUXDB_ADMIN_USER=admin -e INFLUXDB_ADMIN_PASSWORD=supersecretpassword \
       -e INFLUXDB_USER=telegraf -e INFLUXDB_USER_PASSWORD=secretpassword \
       -v $PWD:/var/lib/influxdb \
-      influxdb /init-influxdb.sh
+      influxdb:1.8 /init-influxdb.sh
 ```
 
 The above would create the database `db0`, create an admin user with the password `supersecretpassword`, then create the `telegraf` user with your telegraf's secret password. It would then exit and leave behind any files it created in the volume that you mounted.
 
-# Image Variants
```

Comment:
Not sure why this is deleted (might have messed up running the tool that generates this file)